src/System.Collections/src/Resources/Strings.resx

```diff
@@ -202,7 +193,7 @@
     <value>Target array type is not compatible with the type of items in the collection.</value>
   </data>
   <data name="ArgumentOutOfRange_BiggerThanCollection" xml:space="preserve">
-    <value>Larger than collection size.</value>
+    <value>Must be less than or equal to the size of the collection.</value>
```

Comment:
I fixed the 1st one, not really sure about the 2nd though. Is "Argument_InvalidOffLen" - "Offset and length were out of bounds for the array or count is greater than the number of elements from index to the end of the source collection."